x/campaign/keeper/invariants_test.go

```diff
@@ -11,52 +11,52 @@ import (
 )
 
 func TestAccountWithoutCampaignInvariant(t *testing.T) {
-	k, _, _, _, _, _, ctx := setupMsgServer(t) //nolint
+	ctx, tk, _, _ := setupMsgServer(t) //nolint
```

Comment:
In each module, we have a `setupMsgServer` that returns the message server for the necessary keepers like:
```
keeper.NewMsgServerImpl(*tk.CampaignKeeper),
profilekeeper.NewMsgServerImpl(*tk.ProfileKeeper)
```
Maybe for further simplification we can also add these msgServers in `TestKeepers` as well:
```
type TestKeepers struct {
	CampaignKeeper           *campaignkeeper.Keeper
	LaunchKeeper             *launchkeeper.Keeper
	ProfileKeeper            *profilekeeper.Keeper
	RewardKeeper             *rewardkeeper.Keeper
	MonitoringConsumerKeeper *monitoringcmodulekeeper.Keeper
	BankKeeper               bankkeeper.Keeper
	IBCKeeper                *ibckeeper.Keeper
        LaunchMsgServer    launchkeeper.MsgServer
        ...
}
```